crates/lib/ash-egui/src/egui.frag

```diff
@@ -0,0 +1,11 @@
+#version 450
+
+layout(location = 0) in vec4 inColor;
+layout(location = 1) in vec2 inUV;
+
+layout(location = 0) out vec4 outColor;
+
+layout(binding = 0, set = 0) uniform sampler2D font_texture;
+
+void main() { outColor = inColor.rgba * texture(font_texture, inUV); }
```

Comment:
This would allow us to use a single-component texture -- or do you have plans for using the other components?

```suggestion
void main() { outColor = inColor.rgba * texture(font_texture, inUV).r; }
```

Reply:
No plans?  Is it wasteful to leave the other components unused?